config/schedule.rb

```diff
@@ -31,6 +31,10 @@ def local(time)
   runner "MarkPatientMobileNumbers.call"
 end
 
+every :day, at: local("12:30am"), roles: [:cron] do
```

Comment:
What's this change? The `refresh_materialized_views` task is still under the `db` namespace right? This looks duplicated from line#26 above, but without the ns.

Reply:
This was probably accidental from a merge, I'll double check and clean up.